pkg/services/queryhistory/models.go

```diff
@@ -98,3 +87,30 @@ type QueryHistoryMigrationResponse struct {
 	TotalCount   int    `json:"totalCount"`
 	StarredCount int    `json:"starredCount"`
 }
+
+// CreateQueryInQueryHistoryCommand is the command for adding query history
+// swagger:model
+type CreateQueryInQueryHistoryCommand struct {
+	// UID of the data source for which are queries stored.
+	// example: PE1C5CBDA0504A6A3
+	DatasourceUID string `json:"datasourceUid"`
+	// The JSON model of queries.
+	// required: true
+	// example: [ { "datasourceUid": "PE1C5CBDA0504A6A3", "queries": [ { "refId": "A", "key": "Q-87fed8e3-62ba-4eb2-8d2a-4129979bb4de-0", "scenarioId": "csv_content", "datasource": { "type": "testdata", "uid": "PD8C576611E62080A" } } ], "starred": false, "createdAt": 1643630762, "comment": "debugging" } ]
+	Queries *simplejson.Json `json:"queries"`
```

Comment:
the example does not render as expected:
<img width="1224" alt="Screenshot 2022-06-10 at 1 19 46 PM" src="https://user-images.githubusercontent.com/1632407/173045285-731416c6-ac4c-4c49-a2df-3fa8a1518004.png">
I think you need to define a new type like:
`type Queries *simplejson.Json`
and use this inside the struct (that works for me but I don't know why)

Reply:
Thanks for suggestion! I have updated it in https://github.com/grafana/grafana/pull/50498/commits/b1eb073fbe26fd41351226dde7bdca33838c78f7

Reply:
example is rendered properly now 👍